Core.IntegrationTests/Jira/JiraCombinedIntegrationTests.cs

```diff
@@ -421,6 +423,53 @@ public void FindAllVersionsWithVersionNumber_With3MatchingVersions_Finds3Matchin
     JiraTestUtility.DeleteVersionsIfExistent(_config.Jira.JiraProjectKey, _restClient, fullVersionName, alphaVersionName, betaVersionName);
   }
 
+  [Test]
+  public void AddFixVersionToIssues_WithExistentFixVersions_AddsFixVersionToIssues ()
+  {
+    var fullVersionName = "1.3.0";
+    var alphaVersionName = "1.3.0-alpha.1";
+    var betaVersionName = "1.3.0-beta.3";
+
+    JiraTestUtility.DeleteVersionsIfExistent(_config.Jira.JiraProjectKey, _restClient, fullVersionName, alphaVersionName, betaVersionName);
+
+    var fullVersionID = JiraTestUtility.CreateVersion(_restClient, fullVersionName, _config.Jira.JiraProjectKey);
+    var alphaVersionID = JiraTestUtility.CreateVersion(_restClient, alphaVersionName, _config.Jira.JiraProjectKey);
+    var betaVersionID = JiraTestUtility.CreateVersion(_restClient, betaVersionName, _config.Jira.JiraProjectKey);
+
+    var closedIssueOnlyFull = JiraTestUtility.AddTestIssueToVersion(
+        "Only full version fix",
+        true,
+        _config.Jira.JiraProjectKey,
+        _restClient,
+        fullVersionID);
+    var closedIssueFull1 = JiraTestUtility.AddTestIssueToVersion(
+        "test1",
+        true,
+        _config.Jira.JiraProjectKey,
+        _restClient,
+        alphaVersionID,
+        fullVersionID);
+
+    var outputIssues = _issueService.FindAllClosedIssues(fullVersionID)
+        .ToArray();
+
+    var equivalentIssueIDs= new [] { closedIssueFull1.ID, closedIssueOnlyFull.ID };
+    Assert.That(outputIssues.Select(i => i.ID), Is.EquivalentTo(equivalentIssueIDs));
+
+    _issueService.AddFixVersionToIssues(outputIssues, betaVersionID);
+
+    outputIssues = _issueService.FindAllClosedIssues(fullVersionID)
+        .ToArray();
+
+    Assert.That(outputIssues.All(i => i.Fields.FixVersions.Exists(v => v.ID == betaVersionID)));
```

Comment:
We should add an assertion that both issues also have the full-version. Here we are in integration tests so we test that the entire related stuff is correct. We should also add an unrelated version to the updated issue to document / test that those remain unaffected.